packages/drivers/odsp-driver/src/odspDriverUrlResolver.ts

```diff
@@ -184,6 +187,7 @@ export async function resolveRequest(request: IRequest): Promise<IOdspResolvedUr
         driveId,
         itemId,
         fileName: "",
+        fileExtension: "",
```

Comment:
This should be either .fluid or undefined.
We should not use "" as a special thing - we should support any extension, including no extension

Reply:
Used undefined.